### ISSUE TITLE
Switch Os's and using USB Hards.

### ISSUE THREAD

**Issue by lstew452a** [procount/pinn#609]
How do you switch from one OS to another? 

Also can use this on a hard drive?

**Comment by procount** [comment]
Yes, you can use PINN on a hard drive or SSD. I imagine it is >32GB, in which case, please see https://forums.raspberrypi.com/viewtopic.php?p=1638508#p1638508 in order to install it easily.

The easiest way to switch from one OS to another is to reboot from one OS into PINN, then select the 2nd OS, but are you asking how to switch directly from one OS to the other without going though PINN? This depends on the model of RPi you have. Early Pi4s cannot do this, but see https://github.com/procount/pinn/issues/188
